types/react-big-calendar/index.d.ts

```diff
@@ -31,11 +31,11 @@ export type stringOrDate = string | Date;
 export type ViewKey = 'MONTH' | 'WEEK' | 'WORK_WEEK' | 'DAY' | 'AGENDA';
 export type View = 'month' | 'week' | 'work_week' | 'day' | 'agenda';
 export type ViewsProps = View[] | {
-    work_week?: boolean | React.ComponentType<any> & ViewStatic,
-    day?: boolean | React.ComponentType<any> & ViewStatic,
-    agenda?: boolean | React.ComponentType<any> & ViewStatic,
-    month?: boolean | React.ComponentType<any> & ViewStatic,
-    week?: boolean | React.ComponentType<any> & ViewStatic
```

Comment:
This should be reverted. Otherwise the error message will include a lot of unnecessary types for the props. Since we really don't care about the props here we can use `any` to make the error message more succinct. See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43215

Reply:
I thought `ComponentType<any>` was redundant, reverted it.

One question, I want to understand this: if `ComponentType<any>` must be used for "any component with any props", what's`ComponentType` (without generic parameter, resulting in `ComponentType<{}>` due to generic default value) used for?